base/regex.jl

```diff
@@ -66,14 +66,14 @@ function compile(regex::Regex)
     if regex.regex == C_NULL
         if PCRE.PCRE_COMPILE_LOCK === nothing
             regex.regex = PCRE.compile(regex.pattern, regex.compile_options)
-            PCRE.jit_compile(regex.regex)
+            # PCRE.jit_compile(regex.regex)
```

Comment:
Should depend on the sanitizer option?

Reply:
I need to add the function that exposes it at runtime for it.